compiler/il/ILProps.hpp

```diff
@@ -210,7 +210,7 @@ namespace ILProp2
       ResolveCheck                 = 0x00000040,
       BndCheck                     = 0x00000080,
       CheckCast                    = 0x00000100,
-      // Available                 = 0x00000200,
+      ReadBarrierLoad              = 0x00000200,
```

Comment:
Why do we need this exactly? Isn't this implied by the opcode?

Reply:
It's for later changes in optimizer when we want to special case for this entire group of read barrier opcodes. In dead tree elimination for example, we need to keep the anchoring tree top if it's rdbars. This also mirrors the WriteBarrierStore we already have.